pkgdb/include/flox/buildenv/realise.hh

```diff
@@ -30,6 +30,12 @@
 
 namespace flox::buildenv {
 
+/* -------------------------------------------------------------------------- */
+
+const std::string ACTIVATION_SUBDIR_NAME = "activate";
```

Comment:
There's a circular import that doesn't allow this to be changed to `const char *` and since this is a nonfunctional change I'd rather just submit a followup PR slapping a `// NOLINT(cert-err-58-cpp)` on this line.